docusaurus/docs-internals/Icons/icons.md

```diff
@@ -1 +1,116 @@
+---
+toc_max_heading_level: 4
+---
 # Icons
+
+import { SvgComponent } from '@site/src/components/SvgComponent';
+
+#### Usage
+To specify which `icon` you want to include in the control, pass the icon's name to the icon parameter in the control definition.
+
+```ts
+{
+  id: "optionId",
+  type: "optionType",
+  icon: "nc-settings"
+}
+```
+
+Each icon has a unique name, and this name should be provided as the value for the icon parameter. Make sure to use the correct icon name from the available icon set below to ensure the desired icon is displayed in the control.
+
+### List of available icons
+
+|                                                  |                                                   |                                                   |                                                  |                                                   |
```

Comment:
try to hide this empty row with CSS
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/7c3758ee-f7a1-4b3b-9552-615b1373c331)